composer.json

```diff
@@ -17,7 +17,12 @@
         "rebelcode/module-iterator-abstract": "dev-develop",
         "rebelcode/module-locator": "dev-develop",
         "rebelcode/module-config-interface": "dev-develop",
-        "rebelcode/module-config-validator": "dev-develop"
+        "rebelcode/module-config-validator": "dev-develop",
+        "rebelcode/module-factory-abstract": "dev-develop",
+        "dhii/data-key-value-aware-trait": "dev-task/initial-traits",
+        "dhii/callback-aware-trait": "dev-task/initial-traits",
+        "dhii/config-aware-trait": "dev-task/initial-traits",
```

Comment:
Ideally, they deserve to be in the namespace and package that addresses configuration. There's no such namespace or package currently. Therefore, just here is fine for now. It's not like there are dozens of classes that need `_setConfig()`. Later, when there is a Config lib, we can change that.